applications/serial_lte_modem/doc/ICMP_AT_commands.rst

```diff
@@ -0,0 +1,81 @@
+.. _SLM_AT_ICMP:
+
+ICMP AT Commands
+****************
+
+The following commands list contains ICMP related AT commands:
+
+* `ICMP echo request #XPING`_
+
+ICMP echo request #XPING
+========================
+
+The ``#XGPS`` command sends an ICMP Echo Request, also known as *Ping*.
+
```

Comment:
#GPS does not send ICMP ehco, #ICMP does.

Reply:
#XPING :)